linodecli/exit_codes.py

```diff
@@ -0,0 +1,16 @@
+"""
+This is an enumeration of the various exit codes in Linode CLI
+"""
+
+from enum import Enum
+
+class ExitCodes(Enum):
```

Comment:
We can use the `IntEnum` base class here to avoid needing to get the underlying value of an enum using `.value` 👍

Reply:
Fixed!